### ISSUE TITLE
断言 ！  继承STCollectionView去扩展，这样扩展性不高呀。

### ISSUE THREAD

**Issue by CoderYangFeiLong** [Roylee-ML/SwipeTableView#86]
The item view from dataSouce must be kind of UITalbeView class or STCollectionView class!

**Comment by Roylee-ML** [comment]
Yes. If you use the swipe with mode 1, the item view must not be the class of UICollcetionView or UITableView, otherwise you must use table of stcollection.
